### PR TITLE
Add helpers

### PR DESCRIPTION
**Pull request by cvalarida**
I added these originally in the burials e2e branch, but wanted to get them out before that branch was merged so we can use them in the pensions form.

### CODE REVIEW

File: test/util/nightwatch-commands/fillAddress.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Fills the address form elements.
+ */
+exports.command = function fillAddress(schemaformBaseName, address) {
```
Comment (jbalboni) [inline test/util/nightwatch-commands/fillAddress.js]: I think we could just call `schemaformBaseName` `baseName`.

**Review by jbalboni — APPROVED** [review]

**Review by cehsu — APPROVED** [review]

File: test/util/nightwatch-commands/fillAddress.js (hunk 1 above)
Comment (cvalarida) [inline test/util/nightwatch-commands/fillAddress.js]: Yeah, I originally had it as `baseName`, but I wanted it to be obvious that it was supposed to be `root_parentField_childField`. If it's still obvious, I can shorten it back down to `baseName`.

File: test/util/nightwatch-commands/selectRadio.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/**
+ * Selects the appropriate option for yesNo widgets.
+ *
+ * @param {String} fieldName The name of the field without Yes or No
```
Comment (cehsu) [inline test/util/nightwatch-commands/selectRadio.js]: Love this JSDoc style comment, should we standardize this to automate api documentation?

Comment (cvalarida) [inline test/util/nightwatch-commands/selectRadio.js]: Sounds like a topic to suggest for our FE meeting, eh? 😉

**Pull request merged**